adapters/src/main/kotlin/com/krzykrucz/elesson/currentlesson/monolith/Database.kt

```diff
@@ -1,18 +1,23 @@
 package com.krzykrucz.elesson.currentlesson.monolith
 
+import arrow.core.Option
 import com.krzykrucz.elesson.currentlesson.attendance.domain.Attendance
 import com.krzykrucz.elesson.currentlesson.attendance.domain.IncompleteAttendanceList
+import com.krzykrucz.elesson.currentlesson.lessonprogress.usecase.LessonStatus
```

Comment:
not sure if this dependency is alright, what about putting additional types which are part of  `PersistentCurrentLesson` in the place as `PersistentCurrentLesson`

Reply:
Yeah, moved it to `shared`